src/osd/OSD.h

```diff
@@ -685,6 +685,21 @@ class OSDService {
     return true;
   }
 
+  void dumps_scrub(Formatter *f) {
+    assert(f != nullptr);
+    Mutex::Locker l(sched_scrub_lock);
+
+    f->open_array_section("sched scrubs");
+    for (const auto &i: sched_scrub_pg) {
+      f->open_object_section("scrub");
+      f->dump_stream("pgid") << i.pgid;
+      f->dump_stream("sched_time") << i.sched_time;
+      f->dump_stream("deadline") << i.deadline;
```

Comment:
both "ceph osd scrub ${osd_num}" and "ceph pg scrub ${pg_num}" force a scrub job.